tests/test_ssl.py

```diff
@@ -10,9 +10,9 @@
 import uuid
 
 from gc import collect, get_referrers
-from errno import ECONNREFUSED, EINPROGRESS, EWOULDBLOCK, EPIPE, ESHUTDOWN
+from errno import EAFNOSUPPORT, ECONNREFUSED, EINPROGRESS, EWOULDBLOCK, EPIPE, ESHUTDOWN
```

Comment:
Eeep, line too long, can you wrap this with `()`

Reply:
Done.